<ui version="4.0" >
 <class>QtAppClass</class>
 <widget class="QDialog" name="QtAppClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>QtApp</string>
  </property>
  <widget class="QTextEdit" name="textInput" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>123</y>
     <width>471</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textLoggerIn" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>160</y>
     <width>581</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title" >
    <string>ServerInfo</string>
   </property>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Host:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textHost" >
    <property name="geometry" >
     <rect>
      <x>60</x>
      <y>20</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>210</x>
      <y>20</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text" >
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textPort" >
    <property name="geometry" >
     <rect>
      <x>250</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton" >
    <property name="geometry" >
     <rect>
      <x>310</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>Set Server</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>490</x>
     <y>0</y>
     <width>101</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title" >
    <string>Operation</string>
   </property>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>79</width>
      <height>120</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <widget class="QPushButton" name="btnConnect" >
       <property name="text" >
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDisconnect" >
       <property name="text" >
        <string>Disconnect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClear" >
       <property name="text" >
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSend" >
       <property name="text" >
        <string>Send</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>70</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title" >
    <string>GroupBox</string>
   </property>
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>17</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>SyncDirectory:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSyncDirectory" >
    <property name="geometry" >
     <rect>
      <x>100</x>
      <y>15</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBrowseSyncdirectory" >
    <property name="geometry" >
     <rect>
      <x>370</x>
      <y>13</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>Browse</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_Test_Put_Get" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>480</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>TestPutGet</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3" >
   <property name="geometry" >
    <rect>
     <x>180</x>
     <y>480</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>do_ls</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditUser" >
   <property name="geometry" >
    <rect>
     <x>70</x>
     <y>450</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPassword" >
   <property name="geometry" >
    <rect>
     <x>242</x>
     <y>450</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4" >
   <property name="geometry" >
    <rect>
     <x>13</x>
     <y>456</y>
     <width>54</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text" >
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5" >
   <property name="geometry" >
    <rect>
     <x>190</x>
     <y>456</y>
     <width>54</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text" >
    <string>pass:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSetAccount" >
   <property name="geometry" >
    <rect>
     <x>360</x>
     <y>450</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRegister" >
   <property name="geometry" >
    <rect>
     <x>440</x>
     <y>450</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Register</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSettings" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>480</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Settings</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonQuit" >
   <property name="geometry" >
    <rect>
     <x>480</x>
     <y>490</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textUrl" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>530</y>
     <width>441</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDecode" >
   <property name="geometry" >
    <rect>
     <x>480</x>
     <y>530</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Decode</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>510</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Make UrlDecode() on the String below :</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
